### PR TITLE
Replace Guzzle by Symfony HttpClient

### PR DESCRIPTION
**Pull request by javiereguiluz**
There's nothing wrong with Guzzle, but standardizing the code on Symfony components makes things easier to maintain.

### CODE REVIEW

File: src/CI/UrlChecker.php [hunk 1]
```diff
@@ -11,21 +11,20 @@
...
     {
-        $httpClient = new Client(['timeout' => 10]);
+        $httpClient = HttpClient::create(['timeout' => 10]);
 
         try {
-            $response = $httpClient->get($url, ['http_errors' => false]);
+            $response = $httpClient->request('GET', $url);
             $statusCode = $response->getStatusCode();
-        } catch (GuzzleException $exception) {
+        } catch (\RuntimeException $e) {
```
Comment (nikophil) [inline src/CI/UrlChecker.php:27]: shouldn't we narrow catch to `\Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface`?

Comment (javiereguiluz) [reply src/CI/UrlChecker.php]: I've changed it to `HttpExceptionInterface` ... if we catch only Transport exception we'd miss Server exception, Redirection exception, etc.

Comment (nikophil) [reply src/CI/UrlChecker.php]: indeed, i meant this one :+1:

**Review by nikophil — APPROVED** [review]
LGTM :+1:

**Review by OskarStark — APPROVED** [review]
Looks good 👍🏻😎

**Pull request merged**